schemas/component/ui.schema

```diff
@@ -0,0 +1,113 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/component/ui.schema",
```

Comment:
>https://raw.githubusercontent.com/microsoft/botframework-sdk/master/schemas/component/ui.schema [](start = 10, length = 95)

Gabo is about to check-in that this would change to: [https://schemas.botframework.com/schemas/component/v1.0/ui.schema](https://schemas.botframework.com/schemas/component/v1.0/ui.schema)
You check it into schemas/component/v1.0/ and it gets mapped to that URL.

Reply:
Ok. I'll update once that has been merged.